index.bs

```diff
@@ -8426,10 +8426,10 @@ script.ArrayLocalValue = {
   value: script.ListLocalValue,
 }
 
-script.DateLocalValue = {
+script.DateLocalValue = (
```

Comment:
Mind explaining why these changes are needed?

Reply:
@whimboo IIUC, they're needed because the `.and` operator is being used incorrectly. It is not intended to be a "generic merge operator," which is how we're using it here.

Reply:
I read through the CDDL spec, and it seems that using rounded brackets is indeed necessary when defining a named subgroup that will be embedded into another group.